packages/components/src/internal/components/domainproperties/assay/AssayPropertiesPanel.tsx

```diff
@@ -234,9 +249,14 @@ class AssayPropertiesPanelImpl extends React.PureComponent<Props & InjectedDomai
                     {this.renderBasicProperties()}
                     {this.renderEditSettings()}
                 </Col>
+
                 <Col xs={12} lg={6}>
                     {!appPropertiesOnly && this.renderImportSettings()}
                 </Col>
+
+                <Col xs={12} lg={6}>
+                    {!appPropertiesOnly && this.renderLinkToStudySettings()}
```

Comment:
this isn't a part of this PR changes, but since it is possible to run LK server without the study module, we should probably add a check for the study module as part of this conditional. See other usages of the calls like `hasModule('Study')`

Reply:
Ah, I see. Since you suggest this isn't part of this PR, shall I open an Issue to keep track of that change?

Reply:
Done (as discussed, added todo comment in code)!